### PR TITLE
add request.cookie() and response.cookies

### PR DESCRIPTION
**Pull request by coderhaoxin**
not sure is it necessary to handle some edge cases.

such as:
```js
// from express res
res.cookie('name', '1', { domain: '.1.com' });
res.cookie('name', '2', { domain: '.2.com' });
```

### CODE REVIEW

**Comment by jonathanong** [comment]
oh shit i forgot about this. @coderhaoxin still looking to merge this?

**Comment by jonathanong** [comment]
rebase and re-assign to me if you are!

**Comment by coderhaoxin** [comment]
> rebase and re-assign to me

done

File: README.md [hunk 1]
```diff
@@ -102,6 +102,10 @@ Set header `If-None-Match`.
 
 Set header `Content-Type`.
 
+#### request().cookie(key, value, options)
+
+Set cookie, use [cookie.serialize()](https://github.com/jshttp/cookie)
```
Comment (jonathanong) [inline README.md]: *uses

File: README.md [hunk 2]
```diff
@@ -146,6 +150,10 @@ Get header `ETag`.
 
 Get header `Last-Modified`.
 
+#### response.cookies
+
+Get cookies
```
Comment (jonathanong) [inline README.md]: example of how it looks?

Comment (coderhaoxin) [inline README.md]: done

**Comment by jonathanong** [comment]
https://github.com/thenables/requisition/commit/757af0f20cb93e5ae0a1787131bbe6bd75bd68ea

**Pull request closed without merging**